hapi-proto/src/main/java/com/hederahashgraph/fee/FileFeeBuilder.java

```diff
@@ -279,22 +279,22 @@ public FeeData getFileContentQueryFeeMatrices(int contentSize, ResponseType resp
 
     /*
      * FileGetContentsQuery QueryHeader Transaction - CryptoTransfer - (will be taken care in
-     * Transaction processing) ResponseType - INT_SIZE FileID - 3 * LONG_SIZE
+     * Transaction processing) ResponseType - INT_SIZE FileID - BASIC_ENTITYID_SIZE
```

Comment:
Can BASIC_ENTITYID_SIZE also be named as BASIC_ENTITY_ID_SIZE? If not can this be renamed to something else as BASIC_ENTITYID_SIZE and BASIC_ENTITY_ID_SIZE look alike

Reply:
Forgot to update the comment. fixing the typo